src/components/ha-sidebar.ts

```diff
@@ -548,6 +557,43 @@ class HaSidebar extends LitElement {
     </a>`;
   }
 
+  private _renderExternalConfiguration() {
+    return html`${!this.hass.user?.is_admin &&
```

Comment:
We should check if we show the config panel, as a admin user could also have disabled the config integration (yes, edge case)

Reply:
it's not possible because `config` is a dependency of `frontend`